For unknown reasons llvm_unreachable() causes a GCC internal compiler error
when cross compiling it for ARM:

/builddir/llvm-3.4/lib/LTO/LTOCodeGenerator.cpp: In member function 'void LTOCodeGenerator::setCodePICModel(lto_codegen_model)':
/builddir/llvm-3.4/lib/LTO/LTOCodeGenerator.cpp:166:1: internal compiler error: in merge_if_block, at ifcvt.c:3193
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://gcc.gnu.org/bugs.html> for instructions.

Comment it out for now as workaround.

--- lib/LTO/LTOCodeGenerator.cpp.orig	2014-01-07 12:14:22.401871342 +0100
+++ lib/LTO/LTOCodeGenerator.cpp	2014-01-07 12:14:44.050019574 +0100
@@ -162,7 +162,7 @@ void LTOCodeGenerator::setCodePICModel(l
     CodeModel = model;
     return;
   }
-  llvm_unreachable("Unknown PIC model!");
+  //llvm_unreachable("Unknown PIC model!");
 }
 
 bool LTOCodeGenerator::writeMergedModules(const char *path,
